SPECS/tensorflow/tensorflow.spec

```diff
@@ -102,6 +140,11 @@ bazel --batch build  --verbose_explanations //tensorflow/tools/pip_package:build
 %{_bindir}/toco
 %{_bindir}/toco_from_protos
 
+%files -n python3-tf-nightly
+%license LICENSE
+%{python3_sitelib}/*
```

Comment:
This seems wrong, since `python3-tensorflow` also provides `%{python3_sitelib}/*`. Please double-check, which Python artifacts should belong to `python3-tf-nightly` and split these artifacts between the packages accordingly. You can then add `Requires: python3-tf-nightly = %{version}-%{release}` to `pythno3-tensorflow`, so that `python3-tensorflow` still provides the same set of features.

Reply:
this is the intended design. 
The reason behind this is as follows:
Some of the run time dependencies of Tensorflow like Keras, Tensboard, and  Tensorflow-estimator need TensorFlow as a build time requirement. The build would not even succeed without TensorFlow itself being installed. The tensorFlow team usages nightly package like I did to satisfy this requirement.